### PR TITLE
PLAT-21697 Add run api tests workflow

### PR DESCRIPTION
**Pull request by pjaybell**

### CODE REVIEW

File: .github/workflows/run_api_tests.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+name: Runs API Tests in Github Actions
+
+on:
+  workflow_call:
+    inputs:
+      command:
+        required: true
+        type: string
+      cocker_image_url:
```
Comment (mackenziefinlay) [inline .github/workflows/run_api_tests.yml:9]: typo!

File: .github/workflows/run_api_tests.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+name: Runs API Tests in Github Actions
+
+on:
+  workflow_call:
+    inputs:
+      command:
+        required: true
+        type: string
```
Comment (mackenziefinlay) [inline .github/workflows/run_api_tests.yml:8]: I don't see `command` being used?

File: .github/workflows/run_api_tests.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      - name: Run the api tests
+        env:
+          API_ACCESS_TOKEN: ${{ secrets.API_ACCESS_TOKEN }}
+          API_MUTATION_ACCESS_TOKEN: ${{ secrets.API_MUTATION_ACCESS_TOKEN }}
+          API_MUTATION_INSTANCE: ${{ secrets.API_MUTATION_INSTANCE }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          NOTIFICATION_SNS_TOPIC: ${{ secrets.NOTIFICATION_SNS_TOPIC }}
+        run: |
+          ./entrypoint.sh ${{ inputs.ui_tests }}
```
Comment (mackenziefinlay) [inline .github/workflows/run_api_tests.yml:85]: Is `${{ inputs.ui_tests }}` supposed to be `${{ inputs.command }}``?

**Comment by gsavage** [comment]
This PR relates to Jira ticket PLAT-21697 - see https://administrate.atlassian.net/browse/PLAT-21697

**Review by snewell92 — APPROVED** [review]

**Pull request merged**